apps/src/tutorialExplorer/filterGroupOrgNames.jsx

```diff
@@ -40,7 +40,11 @@ export default class FilterGroupOrgNames extends React.Component {
   render() {
     return (
       <FilterGroupContainer text={i18n.filterOrgNames()}>
+        <label htmlFor="created-by-dropdown" className="hidden-label">
+          Created By
+        </label>
         <select
+          id="created-by-dropdown"
```

Comment:
I think `id` should be unique for the entire page but we can have more than one of these per page?

Reply:
When do we have more than one per page? Or are you saying we could potentially put more than one on a page?

Reply:
Oh, disregard.  I was forgetting that we only have one of these per page.  That said, maybe the ID could be named for the internal name, rather than by the english label we have used, to be something like `"filter-group-org-names-select"`?